brute_force_detector/README.md

```diff
@@ -46,13 +46,8 @@ Unirec template for output interface is `DETECTION_TIME,WARDEN_TYPE,SRC_IP,PROTO
 * `PROTOCOL` : TCP protocol number
 * `DST_PORT` : Port of the attacked service
 * `EVENT_SCALE` : Scale of the detected event
-* `NOTE` : This field contains (comma is used as separator):
-  1. Total number of targets since start of the attack from both 
-directions separated by a dash  
-  2. Flag if the scan is performed
-  3. Number of suspicious flows from both directions and since 
-last report separated by a dash 
-
+* `NOTE` : This field contains (comma is used as separator):   
+  "I: (list of incoming attacker IPs), O: (list of outgoing attacked IPs)" 
```

Comment:
It is great that this description was updated.
However, it still seems strange... even though the description is more accurate now, who is an attacker and who is under attack? We expected that `SRC_IP` is always an attacker. When there is something on the "list of incoming attacker IPs" , is `SRC_IP` a victim for these IPs?

I think the output of the module should describe who is an attacker and victim more clearly...